packages/web/src/components/OpenSourceDatasets/DatasetTable.tsx

```diff
@@ -54,16 +75,37 @@ export default function DatasetTable(props: DatasetTableProps) {
     ) {
         const fieldContent = item[column?.fieldName as keyof PublicDatasetProps] as string;
         if (!fieldContent) return <>--</>;
+        if (
+            column?.fieldName === DatasetAnnotations.RELATED_PUBLICATON.name &&
+            item?.[DatasetAnnotations.DOI.name as keyof PublicDatasetProps]
+        ) {
+            return (
+                <a
+                    className={classNames(styles.link, styles.doubleLine)}
+                    href={item[DatasetAnnotations.DOI.name as keyof PublicDatasetProps]}
```

Comment:
Since `item` is typed as `PublicaDatasetProps` do we gain from having the name of `doi` be used as a constant here? It seems like it is making `Typescript` work against you.

Reply:
good point, it don't think the typescript pain is worth it here. Will change this